src/features/import/components/ImportDialog/elements/ImportMessageList/ProblemRowsText.tsx

```diff
@@ -18,7 +18,7 @@ const ProblemRowsText: FC<Props> = ({ breakpoint = 8, rows }) => {
         values={{ row: rows[0] }}
       />
     );
-  } else if (rows.length <= breakpoint) {
+  } else if (rows.length <= breakpoint || rows.length - breakpoint == 1) {
```

Comment:
Would this make sense instead? I find it slightly easier to read.

```suggestion
  } else if (rows.length <= (breakpoint + 1)) {
```

Reply:
Yes, I was actually debating whether to put one or the other but it seemed to me that the longer version shows the logic behind the change a bit clearer. But yes, the +1 version is more readable, I can change it to that.